website-next/styles/globals.css

```diff
@@ -29,7 +29,7 @@ a {
     Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-display: fallback;
+  font-display: optional;
```

Comment:
what is the difference? it used to be fallback which means it showed an ugly font first, then the loaded nice font, leading to content shift.

but with optional, does it show empty space first, then loaded font? does this decrease content shift? or increase it because now the entire Title will just pop in to place?

what did you see in your testing @enidemi ?

Reply:
From mdn `optional` means "Gives the font face an extremely small block period and no swap period."
There is no font jumping, it waits a little before showing a font, is uses the loaded font if available, otherwise the fallback.
Locally fonts are loaded very fast, so I always saw the loaded font.

Reply:
oooh I see – so if it fails to load the font in time – tough luck! that means we need to pick and configure a pretty similar fallback font then